### DISCUSSION TITLE
Example for Runtime Metrics?

### DISCUSSION THREAD

**Discussion by ndeepakp** [open-telemetry/opentelemetry-java-instrumentation#6530] [Q&A]
Hi,

I see the runtime-metrics instrumentation. So if I import the library via maven/gradle, just adding the below in my properties file should do?
otel.instrumentation.runtime-metrics.enabled

Or do we have an example which shows how to use this?

**Comment by mateuszrzeszutek** [comment]
Hey @ndeepakp ,
If you're talking about `io.opentelemetry.instrumentation:opentelemetry-runtime-metrics`, you'll have to manually install the metrics you want, passing your own `OpenTelemetry` instance. For example, the [memory pool](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/runtime-metrics/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/MemoryPools.java#L52) metrics:
```java
MemoryPools.register(openTelemetry);
```

**Reply by ndeepakp** [reply]
@mateuszrzeszutek Thanks, that is what I have done.

**Reply by trask** [reply]
hi @ndeepakp, if you can provide a full repro that would probably be the easiest way for us to understand why it is not working for you
